Cargo.toml

```diff
@@ -43,7 +46,7 @@ base64 = "0.21.0"
 itertools = "0.10.5"
 time = { version = "0.3.20", features = ["parsing"] }
 yaque = "0.6.4"
-reqwest = "0.11.17"
+reqwest = { version = "0.11.17", default-features = false, features = ["rustls-tls-native-roots"] }
```

Comment:
Needed because `openssl` is not compatible with `musl` without lots of fiddling, and it's easier to just use `rustls`.